crates/shared_types/src/directive.rs

```diff
@@ -136,6 +155,12 @@ pub struct ManualRefConfig {
     pub target_value: String,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Deserialize, Serialize)]
+pub struct AttrStringListConfig {
+    pub attr_name: String,
+    pub value: String,
```

Comment:
shouldn't this be a `Vec<String>` if we have a list? I feel like we should go ahead and support lists if we are claiming it is a list.

Reply:
Changed to accept `Vec<String>` in 525096d2c8e7dd0fc2228985b2c42cdb754d76d9